gnuradio-runtime/lib/pmt/pmt.cc

```diff
@@ -134,7 +134,7 @@ bool to_bool(pmt_t val)
 //                             Symbols
 ////////////////////////////////////////////////////////////////////////////
 
-static const unsigned int get_symbol_hash_table_size()
+static unsigned int get_symbol_hash_table_size()
```

Comment:
This whole function is… strange. Instead of removing the `const` and leaving it `static`, it should be replaced by `constexpr size_t SYMBOL_HASH_TABLE_SIZE = 8192;` at the beginning of the file, and that should be used in the exact two places this function is used.

Reply:
Done.